protolite-well-known-types/protolite-well-known-types.gradle

```diff
@@ -45,6 +45,7 @@ android {
         minSdkVersion project.minSdkVersion
         versionCode 1
         versionName '1.0'
+	consumerProguardFiles 'proguard-rules.pro'
```

Comment:
Fix indentation.

Reply:
I think all other SDKs have the file named proguard.txt. I think that file name may automagically be picked up. Worth confirming and removing this config